src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -10,20 +10,18 @@
        - We need an "empty" build after the last real repo for prebuilt detection to work - this is that file.
        - If we have a repo that is not in sdk's dependency tree, we can still build it by including it here.  -->
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(DotNetBuildPass)' == '' or '$(DotNetBuildPass)' == '1'">
     <RepositoryReference Include="sdk" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(DotNetBuildPass)' == '2' and '$(TargetOS)' == 'windows' and '$(TargetArchitecture)' == 'x64'">
-    <!-- Building Windows Bundle Installers-->
-    <RepositoryReference Remove="@(RepositoryReference)" />
-    <RepositoryReference Include="sdk" DotNetBuildPass="2" />
-  </ItemGroup>
-
+  <!-- Build the cross-OS DACs. -->
   <ItemGroup Condition="'$(DotNetBuildPass)' == '2' and '$(TargetOS)' == 'windows' and '$(TargetArchitecture)' == 'x86'">
```

Comment:
do you know why we build the DAC in the x86 pass instead of x64?

Reply:
Yes, to load balance between the different buildpass2 verticals. We don't want to put everything into win-x64 BP2